### PR TITLE
[#IOCOM-1200] New external api specs

### PR DESCRIPTION
**Pull request by acuiuli**
#### List of Changes

- [new external api specifications](https://github.com/pagopa/io-functions-service-messages/commit/74b992e6c2a8e7c574b1be9ee12dde64f8d473fc)

#### Motivation and Context

A new specification file has been created to send to external people without exposing fields that they do not need to send because are managed internally

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by Garma00 — DISMISSED** [review]

File: openapi/index_external.yaml [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+      summary: Ping
+      description: Returns function's package name and version
+      tags:
+        - restricted
+      responses:
+        '200':
+          description: Success.
+        '500':
+          description: Internal Server error
+  /notify:
```
Comment (michaeldisaro) [inline openapi/index_external.yaml:38]: we should remove this endpoint from the external api

File: openapi/index_external.yaml [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+            $ref: '#/definitions/NewRCConfigurationPublic'
+          x-examples:
+            text/json: |
+                "name": "string",
+                "description": "string",
+                "has_precondition": "NONE",
+                "disable_lollipop_for": [
+                  "DIXDPZ44E08F367A"
+                ],
+                "is_lollipop_enabled": true,
```
Comment (michaeldisaro) [inline openapi/index_external.yaml:89]:
```suggestion
                "is_lollipop_enabled": false,
```

File: openapi/index_external.yaml [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+      description: Create a new remote-content configuration using the provided payload
+      parameters:
+        - in: body
+          name: body
+          required: true
+          schema:
+            $ref: '#/definitions/NewRCConfigurationPublic'
+          x-examples:
+            text/json: |
+                "name": "string",
```
Comment (michaeldisaro) [inline openapi/index_external.yaml:83]:
```suggestion
                "name": "Descriptive configuration name",
```

File: openapi/index_external.yaml [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+                "description": "string",
```
Comment (michaeldisaro) [inline openapi/index_external.yaml:84]:
```suggestion
                "description": "Configuration description.",
```

Comment (michaeldisaro) [reply openapi/index_external.yaml:84]: in all examples

File: openapi/index_external.yaml (hunk 2 above)
Comment (michaeldisaro) [reply openapi/index_external.yaml:89]: in all examples

**Review by michaeldisaro — CHANGES_REQUESTED** [review]

**Review by michaeldisaro — APPROVED** [review]

File: openapi/index_external.yaml [hunk 5]
```diff
@@ -0,0 +1,467 @@
+swagger: '2.0'
+info:
+  version: 1.8.0
+  title: IO API for service Messages
+  contact:
+    name: PagoPA
+    url: https://forum.italia.it/c/progetto-io
+  x-logo:
+    url: https://io.italia.it/assets/img/io-logo-blue.svg
+  description: >
```
Comment (michaeldisaro) [inline openapi/index_external.yaml:10]: Is this experimental? I don't think so. Maybe we should put here a description.

**Review by michaeldisaro — CHANGES_REQUESTED** [review]

**Review by michaeldisaro — APPROVED** [review]

**Review by Garma00 — APPROVED** [review]

**Pull request merged**